issue_template.md

```diff
@@ -0,0 +1,33 @@
+<!--- If you only want to ask a query/question, feel free to remove the below template.
+      For bugs/issues, please fill as many fields as possible. --->
+
+<!--- Provide a general summary of the issue -->
+
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Current Behavior
+<!--- Tell us what happens instead of the expected behavior -->
+
+## Steps to Reproduce
+<!--- Provide an unambiguous set of steps to reproduce this bug.
+      Include code to reproduce, if relevant. -->
+1.
+2.
+3.
+4.
+
+## Context (Environment)
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+#### Service Fabric Runtime and SDK Version :
+
+#### Operating System :
+
+<!--- Cluster size and mention if dev box or prod/test box --->
+#### Cluster Size :
+
+## Possible Workaround

```

Comment:
We should add cluster version and SDK version fields as well explicitly to ensure ppl mention those.

Reply:
line 25 is : `#### Service Fabric Runtime and SDK Version :`
Is `#### Service Fabric Cluster and SDK Version :` better ?

Reply:
@raunakpandya Is line 25 about what you mean ?